imports/plugins/included/payments-stripe/client/checkout/stripe.js

```diff
@@ -66,7 +66,8 @@ AutoForm.addHooks("stripe-payment-form", {
     const storedCard = cardData.type.charAt(0).toUpperCase() + cardData.type.slice(1) + " " + doc.cardNumber.slice(-4);
     Stripe.authorize(cardData, {
       total: Cart.findOne().cartTotal(),
-      currency: Shops.findOne().currency
+      currency: Shops.findOne().currency,
```

Comment:
Shouldn't it be `SellerShops`? In order to take the currency of the seller shop the product belongs to?